KDOP_tree/include/CGAL/KDOP_tree/KDOP_primitive.h

```diff
@@ -0,0 +1,263 @@
+// Copyright (c) 2019  University of Cambridge (UK), GeometryFactory (France)
```

Comment:
Is anything different from the same class of the AABB tree package? I don't see a bbox, so I guess there is no need to copy/rename, but plwease check again.